### PR TITLE
Added data for WUFO roster

### PR DESCRIPTION
**Pull request by alansun25**
- added pics for all sophomores
- got json data file from google form csv
- dynamically populate each player card's data

Still need to do it for La WUFA, but don't have pics for most of the sophomores.

### CODE REVIEW

File: src/components/Player.jsx [hunk 1]
```diff
@@ -19,14 +19,14 @@ export default function Player(props) {
         <div className="cursor-pointer relative">
             {props.team === "wufo" ? 
               <img 
-                src={require(`../assets/wufopics/roster/${props.name}.jpg`).default}
-                alt={`${props.name}`}
+                src={require(`../assets/wufopics/roster/${props.last.toLowerCase()}.jpg`).default}
```
Comment (ritot) [inline src/components/Player.jsx:22]: Might become a problem in the future if there's ever multiple players with the same last name, but I guess whoever is dealing with that can just make the image names be `props.first-props.last.jpg` or use the jersey number instead.

File: src/assets/wufo-players.js [hunk 2]
```diff
@@ -0,0 +1,209 @@
+export const wufoPlayers = [
+  {
+    "isCaptain": true,
+    "name": "Ben Siciliano",
+    "nickname": "Italian Stallion",
+    "year": "Senior",
+    "number": 13,
+    "position": "Cutter",
+    "hometown": "Queens, NY"
```
Comment (ritot) [inline src/assets/wufo-players.js:9]: I think we should add pronouns for all the players too. There's definitely space for it on the back of the card — I'm thinking sandwiched between `Nickname` and `Year`

**Review by ritot — COMMENTED** [review]
The cards look amazing now that they're not all Alan Sun 🤭 

Super close to merging, could you just add the pronouns in please? Also, Nick's card is misaligned because of his two long nicknames — wondering if we should just remove one or let it be.

Screen Shot 2021-09-05 at 8 02 35 PM

File: src/components/Player.jsx (hunk 1 above)
Comment (alansun25) [reply src/components/Player.jsx:22]: I'll make that change

**Comment by alansun25** [comment]
> The cards look amazing now that they're not all Alan Sun 🤭
> 
> Super close to merging, could you just add the pronouns in please? Also, Nick's card is misaligned because of his two long nicknames — wondering if we should just remove one or let it be.
> 
> Screen Shot 2021-09-05 at 8 02 35 PM

Ah yeah that happened with ben's on my screen. I think keeping just Kerosene is fine. And just "Scar" for oscar as well

**Review by ritot — APPROVED** [review]
Looks good to me! ✅

**Pull request merged**